mne/tests/common.py

```diff
@@ -46,3 +47,17 @@ def assert_meg_snr(actual, desired, min_tol, med_tol=500., msg=None):
     snr = np.median(snrs)
     assert_true(snr >= med_tol, 'SNR median %0.2f < %0.2f%s'
                 % (snr, med_tol, msg))
+
+
+def assert_dig_allclose(info_py, info_bin):
+    # test dig positions
+    dig_py = info_py['dig']
+    dig_bin = info_bin['dig']
+    assert_equal(len(dig_py), len(dig_bin))
```

Comment:
@teonlamont let me know where I can grab those text files. It looks like the currently subselected points were likely just the first N points, since they all seem to be in the same "area" (assuming they are supposed to be approximately zero-centered in each coordinate axis). I'll probably update our file to use a regularly or randomly subsampled version instead, and document it somewhere.